source/systems/gaea_user_guide.rst

```diff
@@ -1290,6 +1345,210 @@ For SHMEM codes, users must load the xt-shmem module before compiling:
 
   $ module load xt-shmem
 
+Debugging with DDT
+==================
+
+Arm DDT is a powerful, easy-to-use graphical debugger. With Arm DDT it
+is possible to debug:
+
+* Single process and multithreaded software
+* Parallel (MPI) Software
+* * OpenMP
+* * Hybrid codes mixing paradigms such as MPI + OpenMP, or MPI + CUDA
+
+Arm DDT Supports:
+* C, C++, and all derivatives of Fortran, including Fortran 90.
+* Python (CPython 2.7), limited support
+* Parallel languages/models, including MPI, UPCm and Fortran
+2008 Co-arrays.
+* GPU languages such as HMPP, OpenMP Accelerators, CUDA and
+CUDA Fortran.
+
+
+Arm DDT helps you to find and fix problems on a single thread or
+across hundreds of thousands of threads. It includes static analysis
+to highlight potential code problems, integrated memory debugging to
+identify reads and writes that are outside of array bounds, and
+integration with MPI message queues.
+
+Using DDT on C5
+---------------
+
+To set up the debugger for c5, you must load forge and start ddt on a
+login node, then initiate an interactive session and reverse connect
+from there.
+
+Launch via Login Node
+^^^^^^^^^^^^^^^^^^^^^
+
+In a C5 login node:
+
+.. code-block:: shell
+
+  module load forge/23.1.1
+  ddt
+
+Setup and Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Click the Remote Launch dropdown -> Configure
+2. In the Configure Remote Connections, click Add.
+3. Enter the details of your remote host:
+
+   * Connection Name: Enter a name for your remote connection (otherwise
+     the host name will be used)
+   * Host Name: your_workstation_username@gaea-c5.ncrc.gov
+   * Remote Installation Directory: comes from which ddt on gaea
+   * DDT path for Remote installation: use ` which ddt ` on gaea to find
+     appropriate path'''
```

Comment:
Should the `'''` be here?  (There doesn't seem to be a start for this block...)

Reply:
It should not.  I found that the command:

```bash
sh -c 'dirname $(dirname $(command -v ddt))'
```

will return the correct path required (as indicated in the step) for all shells.  The single quite (`'`) is required.

Reply:
nope. removed.